src/WebJobs.Script/Binding/QueueBinding.cs

```diff
@@ -37,13 +37,16 @@ public override bool HasBindingParameters
             }
 
             boundQueueName = Resolve(boundQueueName);
+            
```

Comment:
That's correct. I wanted to chat with you about the approach before moving forward with the other bindings, but now that we had a chance to talk, I can modify the other bindings to use that information as well.